website/templates/emails/pending_retraction_non_admin.txt.mako

```diff
@@ -0,0 +1,7 @@
+Hello ${user.fullname},
+
+I just wanted to let you know, ${initiated_by} has requested a retraction for the following registration: ${registration_link}.
+
```

Comment:
Yes, you can use the text from one of the other emails explaining what a retraction does, then add as a last paragraph:

"No action is needed from you. Only project administrators can initiate or cancel a retraction.  You are not an administrator on this project.  If you wish to discuss the retraction, contact the administrator(s) of this project."

[Aside: I recall seeing a comment that the person who initiates a retraction should not get the same approve/disapprove email for the retraction.  At first I agreed, but after thinking more we probably want the same behavior to occur as a final confirmation in case someone has second thoughts.  Retracting is a big deal.]